### DISCUSSION TITLE
Newest Priconne patch breaking translations

### DISCUSSION THREAD

**Discussion by digster23** [ImaterialC/PriconeTL#32] [Q&A] [answered]
With the newest patch breaking translations, will they be available again in the future? Is it a relatively easy fix or would the whole game have to be re-translated?

**Comment by ImaterialC** [comment, marked as answer]
currently, we looking for the right framework,
and even if we manage to do it right, 
we still make sure the framework is safe enough to redistribute

**Reply by kylestanley28** [reply]
We just migrated to jp, i really love your tl, keep it up!
